assets/scss/hero.scss

```diff
@@ -82,3 +82,11 @@
     }
   }
 }
+
+.exhibition-page {
+  article, section p {
```

Comment:
Only the paragraphs are now 45rem wide max. Is that indeed correct? The alignment with titles, for example, is a bit off. I will add a screenshot to the ticket.